scilpy/stats/stats.py

```diff
@@ -1,12 +1,60 @@
 # -*- coding: utf-8 -*-
 
-import logging
-
 from itertools import combinations
+import logging
 
+from dipy.denoise.noise_estimate import estimate_sigma, piesno
+import numpy as np
 import scipy.stats
 
 
+def estimate_piesno_sigma(data, number_coils=0):
```

Comment:
In `scilpy.image` we have:
- `volume_math`: that's strictly the stuff from the script volume_math
- `volume_operations`: it could go here, but it's not really an operation?

Or I create a `scilpy.image.volume_metrics`?

Reply:
scilpy.image.volume_metrics fits better.